python/GafferSceneUI/CameraUI.py

```diff
@@ -46,59 +49,237 @@
+
+		"plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
+	
+	],
+
+	"shutter" : [
+
+		"description",
+
+		"""
+		The interval over which the camera shutter is open.	Measured
+		in frames, and specified relative to the frame being rendered.
+		""",
+
+	],
+
+	"resolution" : [
+
+		"description",
+
+		"""
+		The resolution of the image to be rendered.
+		""",
+
+	],
+
+	"pixelAspectRatio" : [
+
+		"description",
+
+		"""
+		The `width / height` aspect ratio of the individual pixels in
+		the rendered image.
+		""",
+
+	],
+
+	"resolutionMultiplier" : [
+
+		"description",
+
+		"""
+		Multiplies the resolution of the render by this amount.
+		""",
+
+	],
+
+	"overscan" : [
+
+		"description",
+
+		"""
+		Whether to enable overscan, which adds extra pixels to the
+		sides of the rendered image.
+		<br><br>
+
+		Overscan can be useful when camera shake or blur will be added
+		as a post-process. This plug just enables overscan as a whole –
+		use the overscanTop, overscanBottom, overscanLeft and
+		overscanRight plugs to specify the amount of overscan on each
+		side of the image.
+		""",
+
+	],
+
+	"overscanTop" : [
+
+		"description",
+
+		"""
+		The amount of overscan at the top of the image. Specified as a
+		0-1 proportion of the original image height.
+		""",
+
+	],
+
+	"overscanBottom" : [
+
+		"description",
+
+		"""
+		The amount of overscan at the bottom of the image. Specified as
+		a 0-1 proportion of the original image height.
+		""",
+
+	],
+
+	"overscanLeft" : [
 
-	"preset:Horizontal", IECoreScene.Camera.FilmFit.Horizontal,
-	"preset:Vertical", IECoreScene.Camera.FilmFit.Vertical,
-	"preset:Fit", IECoreScene.Camera.FilmFit.Fit,
-	"preset:Fill", IECoreScene.Camera.FilmFit.Fill,
-	"preset:Distort", IECoreScene.Camera.FilmFit.Distort,
+		"description",
 
-	"plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
-]
+		"""
+		The amount of overscan at the left of the image. Specified as a
+		0-1 proportion of the original image height.
+		""",
+
+	],
+
+	"overscanRight" : [
+
+		"description",
+
+		"""
+		The amount of overscan at the right of the image. Specified as
+		a 0-1 proportion of the original image height.
+		""",
+
+	],
+
+	"cropWindow" : [
+
+		"description",
+
+		"""
+		Limits the render to a region of the image. The rendered image
+		will have the same resolution as usual, but areas outside the
+		crop will be rendered black. Coordinates range from (0,0) at
+		the top-left of the image to (1,1) at the bottom-right. The
+		crop window tool in the viewer may be used to set this
+		interactively.
+		""",
+
+	],
+
+	"depthOfField" : [
+
+		"description",
+
+		"""
+		Whether to render with depth of field. To ensure the effect is
+		visible, you must also set an f-stop value greater than 0 on
+		this camera.
+		""",
+
+	],
+
+}
 
 ##########################################################################
 # Metadata
 ##########################################################################
 
+__overridesMetadata = {}
```

Comment:
This to L270 should probably be looked at by someone more experienced in Python than I.